### PR TITLE
[scalability] Adjust the threshold for small workloads

### PR DESCRIPTION
**Pull request by mimowo**
#### What type of PR is this?

/kind flake

#### What this PR does / why we need it:

Because the test flaked on the main branch: https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/periodic-kueue-test-scheduling-perf-main/1794363845955817472

Dashboard:
image

Message:
```
=== Failed
=== FAIL: test/performance/scheduler/checker TestScalability/WorkloadClasses/small (0.00s)
    checker_test.go:117: Average wait for admission 227486ms is more then expected 227000ms
```

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

From the comment https://github.com/kubernetes-sigs/kueue/pull/2290#discussion_r1616738204, we also observed 231068ms.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: test/performance/scheduler/default_rangespec.yaml [hunk 1]
```diff
@@ -25,5 +25,5 @@ wlClassesMaxAvgTimeToAdmissionMs:
   # Average value 76768 (+/- 2%), setting at +20%
   medium: 90_000
 
-  # Average value 215468 (+/- 2%), setting at +5%
-  small: 227_000
+  # Average value 215468 (+/- 2%), setting at +5% + 3s accounting for observed failures
+  small: 230_000
```
Comment (tenzen-y) [inline test/performance/scheduler/default_rangespec.yaml:29]: Is this enough?
Actually, we observed the 231068ms here: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-kueue-test-scheduling-perf-main/1795196252221083648

Comment (mimowo) [reply test/performance/scheduler/default_rangespec.yaml:29]: Oh I didn't know about the other failure, then maybe 233_000, to have some buffer?

**Review by tenzen-y — COMMENTED** [review]
Thanks!
/lgtm

**Pull request merged**